Header header
int32 x_size
int32 y_size
int32 z_size
float32 x_origin
float32 y_origin
float32 z_origin
float32 width
float32 x_carrot
float32 y_carrot
float32 z_carrot
uint8 TYPE_OCCUPANCY=0
uint8 TYPE_EDT=1
uint8 TYPE_NF1=2
uint8 type
uint8[] payload8